### PR TITLE
Prepare iota-client 1.0.0-rc.1 Java release

### LINKED ISSUE #1411: Add documentation on how to start with Android

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/iota.rs/issues/1411

### CODE REVIEW

**Review by thibault-martinez — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

**Comment by kwek20** [comment]
https://github.com/iotaledger/iota.rs/blob/3e9b5c3bee73ed6c14b9cacde548cad9698e27ca/client/bindings/java/lib/native/Cargo.toml#L4-L5

name and version maybe still need a change?

**Comment by thibault-martinez** [comment]
> https://github.com/iotaledger/iota.rs/blob/3e9b5c3bee73ed6c14b9cacde548cad9698e27ca/client/bindings/java/lib/native/Cargo.toml#L4-L5
> 
> name and version maybe still need a change?

If i'm correct, this is only the version of the rust glue code, but it ends up never being published as is, only the jar is.

File: client/bindings/java/README.md [hunk 1]
```diff
@@ -13,59 +13,62 @@ keywords:
...
-## Requirements
+## Use in your Android project (Android Studio)
 
-* Make sure you have the latest [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/downloads/) installed.
+1. Add following dependency to your `build.gradle` file:
+```
+implementation 'org.iota:iota-client:1.0.0-rc.2'
+```
+
+2. Download the `org.iota:iota-client:1.0.0-rc.1-jni-libs.zip` file from the Github release and unzip it.
```
Comment (Thoralf-M) [inline client/bindings/java/README.md]: 1.0.0-rc.2
1.0.0-rc.1

File: documentation/docs/getting_started/java.mdx [hunk 2]
```diff
@@ -4,74 +4,77 @@ image: /img/logo/iota_mark_light.png
...
 
 Get started with the official IOTA Client Java Library.
 
-## Security
+## Use in your Android project (Android Studio)
 
-<WarningPasswordStorage/>
+1. Add following dependency to your `build.gradle` file:
+```
+implementation 'org.iota:iota-client:1.0.0-rc.2'
```
Comment (Thoralf-M) [inline documentation/docs/getting_started/java.mdx]: Here should also be .1?

File: client/bindings/java/README.md (hunk 1 above)
Comment (samuel-rufi) [reply client/bindings/java/README.md]: yes

File: documentation/docs/getting_started/java.mdx (hunk 2 above)
Comment (samuel-rufi) [reply documentation/docs/getting_started/java.mdx]: yes

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**